src/test/resources/domain/GET/max-recursions.yml

```diff
@@ -0,0 +1,12 @@
+step_1:
+  assign:
+    stringValue: ""
+
+step_2:
+  assign:
+    stringValue: ${stringValue + "test"}
+  maxRecursions: 5
```

Comment:
1. If `maxRecursions` in `application.yml` is set to `10`, running this step `5` times is OK;
2. If `maxRecursions` in `application.yml` is set to `2`, this step should only be run `2` times.